js/pulse.js

```diff
@@ -643,6 +656,10 @@
                         adPlayer.addEventListener(OO.Pulse.AdPlayer.Events.AD_CLICKED, _.bind(_onAdClicked,this));
                         adPlayer.addEventListener(OO.Pulse.AdPlayer.Events.LINEAR_AD_PAUSED, _.bind(_onAdPaused,this));
                         adPlayer.addEventListener(OO.Pulse.AdPlayer.Events.LINEAR_AD_PLAYING, _.bind(_onAdPlaying,this));
+
+                        if(pluginCallbacks.onAdPlayerCreated) {
```

Comment:
Please add an extra null check for pluginCallbacks. You may guarantee it's not null now, but if someone changes the code in the future, it may break during this if check.